schema/user.js

```diff
@@ -17,13 +28,8 @@ const typeDef = gql`
 
 const userResolvers = {
   Query: {
-    user: async (root, {}, { accessToken, dataSources }) => {
-      const userSettings = await dataSources.walloraAPI.getUserSettings(
-        accessToken
-      );
-      return { userSettings };
-    },
-  },
-};
+    user: () => ({ userSettings }),
```

Comment:
fetch data from remote api and then map it to the new format and send back. Do not send back dummy data for user settings